src/components/Dropdown/Dropdown.js

```diff
@@ -79,6 +79,8 @@ const SelectedItem = styled(Item)`
   div:last-child {
     margin-left: auto;
   }
+  ${props =>
+    props.disabled && `background-color: ${props.theme.colors.gray50};`};
 `;
```

Comment:
![dropdown-disabled](https://user-images.githubusercontent.com/25717783/73146503-0b4e8e80-40d9-11ea-93cd-1d9860f9896d.png)
Dropdown disabled with this changes

Reply:
![input-disabled](https://user-images.githubusercontent.com/25717783/73146679-e7d81380-40d9-11ea-9e67-cfcdf419dc59.png)

Input when disabled

Reply:
Perfect, looks much better IMO :)